src/NuGet.Core/NuGet.Packaging/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+~static NuGet.Packaging.Rules.AnalysisResources.MissingReadmeInformation.get -> string
```

Comment:
I feel like we need to redesign this. It's pretty poor that we need to make the analysis resource public.